server/apps/users/views.py

```diff
@@ -52,75 +63,65 @@ def post(self, request):
         response = Response()
         response.set_cookie(key='jwt', value=access_token, httponly=True)
 
-        # Generate JWT refresh token
-        refresh_token = generate_jwt_refresh_token(user)
-        response.set_cookie(key='refresh_token', value=refresh_token, httponly=True)
-
         response.data = {
             'message': AUTHENTICATION_SUCCESSFUL
         }
 
         return response
 
 
-class RefreshTokenView(APIView):
-    # Using @csrf_protect decorator for making sure that the cookie (refresh token) is not compromised
-    csrf_protect_method = method_decorator(csrf_protect)
-
-    @csrf_protect_method
-    def post(self, request):
-        refresh_token = request.COOKIES.get('refresh_token')
-
-        payload = verify_jwt_refresh_token(refresh_token)
+class UserView(generics.RetrieveAPIView):
+    serializer_class = UserSerializer
+    permission_classes = [IsAuthenticated]
```

Comment:
`permission_classes = [IsAuthenticated]`

This you can set as default un settings as most of the APIs will need authetication, so you don't need to provide it everywhere.

Reply:
Updated in latest commit